packages/blocks/src/api/factory.js

```diff
@@ -212,6 +231,42 @@ const getBlockTypesForPossibleToTransforms = ( blocks ) => {
 	return blockNames.map( ( name ) => getBlockType( name ) );
 };
 
+/**
+ * Determines whether transform is a "block" type
+ * and if so whether it is a "wildcard" transform
+ * ie: targets "any" block type
+ *
+ * @param  {Object} t Block transform object
+ * @return {boolean}   whether transform is a wildcard transform
+ */
+const isWildCardBlockTransform = ( t ) => t && t.type === 'block' && t.blocks.length && t.blocks[ 0 ] === '*';
```

Comment:
Minor: Wildcard, as a single word, should be capitalized as `Wildcard`, not `WildCard`

Reply:
👍